variants.go

```diff
@@ -13,6 +37,14 @@ type Variant struct {
 	SKU string
 	// Title is the title of the product variant.
 	Title string
+	// Option1 is a custom properties that a shop owner uses to define product variants. This is option1 of three available.
```

Comment:
Same with the comments below. Probably no need for the 1/3 available. They can see that in the struct.

```suggestion
	// Option1 is a custom property that a shop owner uses to define product variants.
```